docs/.vuepress/styles/index.styl

```diff
@@ -12,10 +12,8 @@ table {
   tr, th, td {
     border: none;
   }
-  td:not(:last-of-type) {
-    border-right: 1px solid var(--grey-200);
-  }
-  tr:nth-child(2n) {
+
+  tr, tr:nth-child(2n) {
```

Comment:
side note: nth-child supports `even` and `odd` https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#keyword_values

bit easier to read

Reply:
It doesn't collide because in here we are trying to override a default vuepress style. 🙄 Using just `tr` doesn't have enough specificity to override and there was no background color by default (which is what I was trying to change).